sentences/de/cover_HassGetState.yaml

```
@@ -4,16 +4,16 @@ intents:
     data:
       - sentences:
           - "ist[ <aktuell>] <name> {cover_states:state}[ <area_floor>]"
-          - "ist <name>[ <aktuell>] {cover_states:state}[ <area_floor>]"
+          - "ist <name> <aktuell> {cover_states:state}[ <area_floor>]"
           - "ist[ <aktuell>] <name>[ <area_floor>] {cover_states:state}"
-          - "ist <name>[ <aktuell>][ <area_floor>] {cover_states:state}"
-          - "ist <name>[ <area_floor>][ <aktuell>] {cover_states:state}"
+          - "ist <name> <aktuell>[ <area_floor>] {cover_states:state}"
+          - "ist <name> <area_floor> <aktuell> {cover_states:state}"
           - "ist[ <aktuell>][ <area_floor>] <name> {cover_states:state}"
-          - "ist[ <area_floor>][ <aktuell>] <name> {cover_states:state}"
+          - "ist[ <area_floor>] <aktuell> <name> {cover_states:state}"
```

```suggestion
          - "ist <area_floor> <aktuell> <name> {cover_states:state}"
```
`ist <aktuell> <name> {cover_states:state}` already covered in line 6